src/main.cpp

```diff
@@ -669,9 +669,15 @@ Args parse_args(int argc, char* argv[])
     if (optind == argc) {
       args.search = "*:*";
     } else if (optind == argc - 1) {
-      args.search = argv[optind];
-      if (args.search == "*") {
+      std::string val(argv[optind]);
+      if (val == "*") {
         args.search = "*:*";
+      } else if (val.find(":") != std::string::npos) {
```

Comment:
I think we can also check if `*` exists in the value and assume it's not a file.